web/themes/custom/customtheme/components/01-atoms/buttons/back-to-top.js

```diff
@@ -0,0 +1,22 @@
+(function ($, Drupal, once) {
+  Drupal.behaviors.backToTop = {
+    attach: function (context, settings) {
+      var backButton = once('backToTopButton', '.font-container', context)[0];
```

Comment:
once(...)[0]
And what if there is no .font-container on a certain page, therefore no index 0 element?
What if there are a few of them? Others are ignored? Probably there will be no more than 1 backToTopButton, but still, it is no effort to support any amount of them

That said, selector ".font-container" doesn't make sense in context of "Back to Top" functionality